src/Compilers/Core/CodeAnalysisTest/Text/StringTextDecodingTests.cs

```diff
@@ -133,7 +133,7 @@ public void CheckSum_SHA256()
             Assert.Equal("f1945cd6 c19e56b3 c1c78943 ef5ec181 16907a4c a1efc40a 57d48ab1 db7adfc5", StringTextTest.ChecksumToHexQuads(checksum));
         }
 
-        [ConditionalFact(typeof(IsEnglishLocal))]
+        [ConditionalFact(typeof(HasEnglishDefaultEncoding))]
         [WorkItem(5663, "https://github.com/dotnet/roslyn/issues/5663")]
         public void Decode_NonUtf8()
```

Comment:
This test failed on my machine, even though both `CurrentCulture` and `CurrentUICulture` returned `un-US`. It seems the default encoding (which is Windows-1250, not Windows-1252, on my machine) is not the same as current culture.